src/WebJobs.Extensions.DurableTask/EventSourceListener.cs

```diff
@@ -39,10 +39,8 @@ public EventSourceListener(LinuxAppServiceLogger logger, bool enableVerbose, End
         /// <param name="eventSource">An instance of EventSource.</param>
         protected override void OnEventSourceCreated(EventSource eventSource)
         {
-            // "7DA4779A-152E-44A2-A6F2-F80D991A5BEE" is the old DurableTask-Core event source,
-            // so we provide extra logic to ignore it.
-            if ((eventSource.Name == "DurableTask-Core"
-                  && eventSource.Guid != new Guid("7DA4779A-152E-44A2-A6F2-F80D991A5BEE")) ||
+
+            if (eventSource.Name == "DurableTask-Core" ||
```

Comment:
So, for the code in `dev`, we explicitly filter out the "DurableTask-Core" provider of GUID "7DA4779A-152E-44A2-A6F2-F80D991A5BEE" since it "is the old DurableTask-Core event source". From my testing, in V1, it is also the _only_ DurableTask-Core event source, so filtering it out means we get no Core logs. Does that ring true to you all?

Reply:
I don't believe it is the only DurableTask-Core implementation, because we update v1's version of DurableTask.AzureStorage to get the latest changes. That will inherently increase our DurableTask.Core version to one that has the new event source.

Reply:
I trust that's correct but somehow my tests were missing logs when I had that filter enabled. I'll add the filter back and try again, it's been a while since I validated this.

Reply:
Yeah, in my tests, the filtere'd out GUID is the only ETW event with name "DurableTask-Core".
I'll sync with you about this internally early next week

Reply:
We need to keep this filter as-is and change our DurableTask.AzureStorage dependency to the latest version.

Reply:
@cgillum, yup, turned out that was the culprit :) . Thanks for pointing that out!